src/shell/hooks/useFilePath.js

```diff
@@ -14,15 +18,31 @@ export function useFilePath(ZUID = "") {
   // There is no place where this method is called by a content item zuid, if there was we would need to reference all items.
   switch (Number(ZUID.split("-")[0])) {
     case zuid.prefix.SITE_CONTENT_SET:
-      let findfile = files.find(file => file.contentModelZUID === ZUID);
+      let findfile = files.find((file) => file.contentModelZUID === ZUID);
       if (findfile) {
         return `/code/file/views/${findfile.ZUID}`;
       } else {
         return "/code";
       }
 
+    // Searching on Content Item Zuid 7-
+    case zuid.prefix.SITE_CONTENT_ITEM:
+      let findItemZUID = Object.keys(content).find((file) => file === ZUID);
+      let findMatch = Object.values(content).filter(
+        (file) => file.meta.ZUID === findItemZUID
+      );
+      let findFile = files.find(
+        (file) => file.contentModelZUID === findMatch[0].meta.contentModelZUID
+      );
+
+      if (ZUID === findItemZUID && findFile) {
```

Comment:
It is only necessary to confirm a file was found. If so we know it's ZUID will be present on that object.